content/docs/internals/troubleshooting.mdx

```diff
@@ -203,6 +203,26 @@ Usually, this is the result of either a routing issue or a configuration error.
 
 <GenerateRecoveryToken />
 
+### Directory sync
+
+**Problem:**
+
+When syncing your identity provider directory to the Enterprise Console, you may have issues completing the sync due to timeouts or errors.
```

Comment:
Would it make sense to phrase the problem more like this?

```suggestion
Enterprise Console directory sync fails to complete.
```

(I see the other troubleshooting sections aren't really set up like Problem / Solution, but I wonder if it would make sense to give a very brief "Problem" summary for the others too.)

Reply:
@kenjenkins yeah I figured I'd try this format, it seemed to fit this scenario well. Maybe we could clean up the existing sections and add this formatting in a separate PR? Could also take your idea to make this a top-level page into account in that PR.

Reply:
Definitely, a separate PR sounds good to me.